doc/xep.doc

```diff
@@ -46,5 +46,6 @@ Ongoing:
 - XEP-0009: Jabber-RPC (API is not finalized yet)
 - XEP-0060: Publish-Subscribe (Only basic IQ implemented)
 - XEP-0077: In-Band Registration (Only basic IQ implemented)
+- XEP-0363: HTTP File Upload [v0.9.0]
```

Comment:
What does the version refer to?

Reply:
It's the XEP's version, so we can easily check if an implementation is outdated.

Reply:
I also added the version number in the header files, maybe that's a bit too much?